Makefile

```diff
@@ -9,7 +9,7 @@ prep: checkout
 	# BlackParrot
 	$(MAKE) -C $(BLACKPARROT_DIR) libs
 	$(MAKE) -C $(BLACKPARROT_TOOLS_DIR) tools
-	$(MAKE) -C $(BLACKPARROT_SDK_DIR) sdk
+	$(MAKE) -C $(BLACKPARROT_SDK_DIR) sdk PLATFORM=zynqparrot
```

Comment:
I think technically this is set in Makefile.common, but doesn't hurt to be explicit

Reply:
ah, yes, I missed that. Modified to be explicit and use the def in Makefile.common